bindata/network/kuryr/alert-rules.yaml

```diff
@@ -121,3 +121,17 @@ spec:
       for: 5m
       labels:
         severity: critical
+    - alert: KuryrLoadBalancerNotReady
+      annotations:
+        message: Kuryr noticed an Octavia load balancer unexpectedly stuck in PENDING_* state in the last 20 minutes. Most likely this indicates an issue with OpenStack Octavia.
```

Comment:
Will it only start alerting after 20 min? It might be too long, maybe 10 mins fits better?

Reply:
It works like that: if there is any change on the counter within 20 minutes timeframe, then alarm will be triggered. Given the fact, we do the reconciliation, and retry loop is taken around 10-15 minutes, then we make sure that alarm will remain because we will re-trigger it within 20 min block. Alarm will go if counter stops increase itself. Initially @dulek proposed 30 minutes for the window, but IMHO it's a little bit to long.

Reply:
@dulek do you still have a preference for 30 min?

Reply:
Not at all, @gryf convinced me to do 20 minutes.